package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "ramda-lens",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
xyz updates `"version"` as part of the release commit, so `"0.0.0"` is the most appropriate value for an unpublished project. If we run `npm run release-minor` we'll release v0.1.0, or if we decide to run `npm run release-major` we'll go straight to v1.0.0.